Sources/EmbraceCaptureService/CaptureService+OTel.swift

```diff
@@ -0,0 +1,111 @@
+//
+//  Copyright © 2025 Embrace Mobile, Inc. All rights reserved.
+//
+
+import Foundation
+
+#if !EMBRACE_COCOAPOD_BUILDING_SDK
+    import EmbraceCommonInternal
+    import EmbraceSemantics
+    import EmbraceConfiguration
+#endif
+
+/// This extension provides functions to generate OTel data within a `CaptureService`.
+extension CaptureService {
+
+    /// Creates a new span to be included in the current Embrace session.
+    /// - Parameters:
+    ///   - name: Name of the span.
+    ///   - parentSpan: Parent of the span, if any.
+    ///   - type: Embrace specific type of the span. Defaults to `.performance`.
+    ///   - status: Initial status of the span. Defaults to `.unset`.
+    ///   - startTime: Start time of the span. Defaults to the current time.
+    ///   - endTime: End time of the span, if any.
+    ///   - events: Events for the span.
+    ///   - links: Links for the span.
+    ///   - attributes: Attributes of the span.
+    ///   - autoTerminationCode: If a code is passed, the span will be automatically ended when the current Embrace session ends and will have a special attribute with the given code.
+    /// - Returns: The newly created `EmbraceSpan`.
+    /// - Throws: `EmbraceOTelError.spanLimitReached` if the span limit has been reached for the current Embrace session.
+    @discardableResult
+    public func createSpan(
```

Comment:
I've always found it confusing that we can create spans from many different places. I've wonder if we might pull it in a bit and make sure there's only one place that it can be done from.